.github/workflows/note-arduino-ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Note Tutorials CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  validate_esp32_examples: # job id
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        include:
+          - sketch: sensor-tutorials/arduino-wiring/nano-no-library
+            fully-qualified-board-name: esp32:esp32:nano32
+            libraries: "https://github.com/Seeed-Studio/Seeed_BME680"
+          - sketch: sensor-tutorials/arduino-wiring/nano-with-library
```

Comment:
This example should be kept, because it consumes/validates the Seeed BME680, which is not covered by any other examples.

_see details below..._